boards/x86/qemu_x86/doc/board.rst

```diff
@@ -63,3 +62,35 @@ The following platform features are unsupported:
 * Inter-Integrated Circuit (I2C)
 * Ethernet
 * Supervisor Mode Execution Protection (SMEP)
+
+Programming and Debugging
+*************************
+
+Use this board configuration to run basic Zephyr applications and kernel tests
+in an emulated enviroments. For example:
+
+.. code-block:: console
+
+        $ make -C samples/synchronization run
+
+This will build an image and boots it using QEMU with the following output:
```

Comment:
This will build an image with the synchronization sample app, boot it using QEMU,
     and display the following console output: